### ISSUE TITLE
RE: sysmonconfig-export.xml  

### ISSUE THREAD

**Issue by zabboto** [SwiftOnSecurity/sysmon-config#134]
Line 335:     <DestinationPort name="Alert,Metasploit" condition="is">444</DestinationPort>

**Issue:**
I noticed that the line above indicates '444' as the default Metaspolit destination port, but I think the default listener is actually '4444'. Let me know if I have misunderstood what this line is indicating.

**Comment by DaGrand** [comment]
That's correct another user open the same thread and include a PR https://github.com/SwiftOnSecurity/sysmon-config/issues/142
Could you close this one please?
